docs/src/languages/nodejs/debug.md

```diff
@@ -3,121 +3,244 @@ title: "Debugging"
+<--->
 
-## Locally
+---
+title=Local
+file=none
+highlight=false
+---
 
-### Create a new environment
+In the same terminal as the previous step, run the following command:
 
-```sh
-$ platform branch debug-branch
+```bash
+node --inspect <START_FILE>
 ```
 
-### Open a tunnel
+Replace `<START_FILE>` with the file defined for [your app's `start` command](./_index.md#usage-example).
+<!--vale off -->
+{{< /codetabs >}}
+<!--vale on -->
+
 
-To get access to your application's relationships locally, open a tunnel to your application container:
+You get output something like this:
 
-```sh
-$ platform tunnel:open
+```bash
+Debugger listening on ws://127.0.0.1:9229/10701e5d-d627-4180-a967-d47a924c93c0
+For help, see: https://nodejs.org/en/docs/inspector
+Listening on port 8888
 ```
 
-In a multi-application project, you will be prompted to chose which application to open a tunnel to. You can also specifiy the application you want by using the `--app` option:
+## 4. (If debugging remotely) Forward the debugger port locally
 
-```sh
-$ platform tunnel:open --app=my-app-name
+In another terminal, create an SSH tunnel that forwards to the 9229 port:
+
+```bash
+ssh -N -L9229:127.0.0.1:9229  $(platform  ssh --pipe)
 ```
 
-### Set the environment variables
+## 5. Connect the debugger
 
-In the terminal in which the tunnel has been opened, run:
+You can now connect the debugger as if you were debugging a local application.
+See examples with some common tools:
 
-```sh
-$ export PLATFORM_RELATIONSHIPS="$(platform tunnel:info --encode)"
-$ export PORT=8888
-```
+{{< codetabs >}}
 
-### Run your application in inspect mode
+---
+title=Using Chrome developer tools
+file=none
+highlight=false
+---
 
-```sh
-$ node --inspect node --inspect <js_file>
-```
+Go to `chrome://inspect`.
+Find your running app under the `Remote Target` list.
+Click **inspect** to start the debugger.
+
+<--->
+
+---
+title=Using Visual Studio Code
+file=none
+highlight=false
+---
+
+Use the `Node.js: Attach` debugger option.
+
+If you haven't created the option:
+
+1. On the **Run and Debug** tab, click `create a launch.json file`.
+2. Select `Node.js` as the environment.
+3. In the `configurations` array, start IntelliSense (usually <kbd>ctrl</kbd>+<kbd>space</kbd>).
+4. Select `Node.js: Attach`.
+5. Make sure the port is the same as in [step 4 above](#4-if-debugging-remotely-forward-the-debugger-port-locally).
+
+Once you have the option:
+
+In the **Run and Debug** tab, select `Attach` from the menu and click **Start Debugging** (the green arrow).
+
```

Comment:
:bulb:

It could be nice to add the reference to [the documentation](https://code.visualstudio.com/docs/nodejs/nodejs-debugging) (same for chrome)?  Since it's the last step of the process, if they leave our documentation it's not an issue.

Reply:
I couldn't find a similar overview for Chrome quickly.

It's not technically the end of the step, so it's potentially a small issue, but those docs should help if they like them.